scripts/start_backer.sh

```diff
@@ -12,7 +15,7 @@ mkdir -p $CONFIG_DIR/keri/cf
 echo '{
     "backer": {
       "dt": "'$(date -u +"%Y-%m-%dT%H:%M:%S.000000+00:00")'",
-      "curls": ["'$URL':'$PORT'/"]
+      "curls": ["tcp://'$HOST':'$PORT'/", "'$URL':'$PORT'/"]
```

Comment:
@rodolfomiranda This works outside of Docker but within the Docker container nothing happens when creating an AID.

Based on `.env.example` - we get `http://cardano-backer:5666` here - is this supposed to be Docket network or external (in my case, localhost)? Neither work for me in Docker. Ports are exposed and mapped directly. (KERIA doesn't complain either)

Reply:
Based on Roberto's old deployment - `tcp://cardano-backer:5665` (ending on 5) and `http://localhost:5666` - this works for me, will push some changes